<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Account::LedgerViewerWidget</class>
 <widget class="QWidget" name="Account::LedgerViewerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ledger</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" colspan="6">
      <widget class="QFrame" name="menuFrame">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>361</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="menuHorizontalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="yearsLabel">
       <property name="text">
        <string>Years</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="buddy">
        <cstring>yearsComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="yearsComboBox"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="monthsLabel">
       <property name="text">
        <string>Months</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="buddy">
        <cstring>monthsComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="6">
      <widget class="QGroupBox" name="receiptsGroupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>275</height>
        </size>
       </property>
       <property name="title">
        <string>Receipts</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTableView" name="tableView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="sumLabel">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Total sum = </string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="4">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="monthsComboBox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
